code/game/machinery/OpTable.dm

```diff
@@ -16,6 +16,7 @@
 	var/mob/living/carbon/human/victim = null
 	var/strapped = 0.0
 	var/obj/machinery/computer/operating/computer = null
+	var/obj/machinery/vitals_monitor/connected_monitor = null
```

Comment:
You don't actually need to do `= null` on var declarations, it's copypasted from other code.